README.md

```diff
@@ -50,6 +50,25 @@ You might want to install freshly if you still use the old version.
 
 # Recent changes:
 
+## 2021-06-08
+
+### Added
+
+- refactoring the plotting system (#166)
+- Add version log message to main ``__init__`` (#175)
+
+### Fixed
+
+- Fix crop if less than one row is not nan (#198)
```

Comment:
Could note that this fixes https://github.com/toolsforexperiments/plottr/issues/196